Hippo.Web/ClientApp/src/components/RequstForm.tsx

```diff
@@ -1,13 +1,17 @@
 import { useContext, useEffect, useState } from "react";
 
 import AppContext from "../Shared/AppContext";
-import { Account } from "../types";
+import { Account, RequestPostModel } from "../types";
 import { authenticatedFetch } from "../util/api";
 
 export const RequestForm = () => {
   const user = useContext(AppContext).user;
 
   const [sponsors, setSponsors] = useState<Account[]>([]);
+  const [request, setRequest] = useState<RequestPostModel>({
+    sponsorId: 0,
```

Comment:
@srkirkland How do I set this to the first sponsor (which the select is defaulted to)